rpm/mlsdb-data.spec

```diff
@@ -46,6 +49,10 @@ Moldova, Poland, Romania, Slovakia and Ukraine.
 %prep
 %setup -q -n %{name}-%{version}
 
+%build
+%qmake5 translations
```

Comment:
Would it be simpler to just call lupdate and copy the generated file in place?

Reply:
Duh. It would be. Fixed.